config/build.yml

```diff
@@ -18,7 +18,7 @@ sources:
 #@ if/end data.values.push_images:
 ---
 apiVersion: kbld.k14s.io/v1alpha1
-kind: ImageDestinations
+kind: Config
```

Comment:
do we want this change?

Reply:
I used this schema suggested by @neil-hickey 

and I checked the doc from https://carvel.dev/kbld/docs/v0.34.0/config/, is it the latest schema?

Reply:
i think it's fine to adopt `Config`